src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.data;
+
+public final class User {
+
+  private final String name;
+  private final String email;
+  private final String wishlist;
+  private final String teachlist;
```

Comment:
If these are supposed to be actual lists then it's better to use an actual list:
```suggestion
  private final List<String> wishList;
  private final List<String> teachList;
```

I've left similar suggestions in other comments